src/main/java/com/eliasnogueira/driver/BrowserFactory.java

```diff
@@ -93,6 +113,14 @@ public WebDriver createLocalDriver() {
             return new EdgeDriver(getOptions());
         }
 
+        public WebDriver createTestContainerDriver() {
+            BrowserWebDriverContainer<?> driverContainer = new BrowserWebDriverContainer<>()
+                    .withCapabilities(new EdgeOptions());
+                driverContainer.start();
+                return new RemoteWebDriver(driverContainer.getSeleniumAddress(), new EdgeOptions());
+
```

Comment:
## Fix

Can you remove the empty line?

Reply:
done